src/pages/Preferences.vue

```diff
@@ -36,8 +36,23 @@ export default Vue.extend({
     });
 
     ipcRenderer.send('api-get-credentials');
+
+    ipcRenderer.on('k8s-integrations', (_, integrations) => {
+      this.$store.dispatch('preferences/setWslIntegrations', integrations);
+    });
+
+    ipcRenderer.send('k8s-integrations');
+
+    this.$store.dispatch('preferences/setPlatformWindows', os.platform().startsWith('win'));
+  },
+  computed: {
+    ...mapGetters('preferences', ['getPreferences', 'isPreferencesDirty', 'isPlatformWindows']),
+    navItems(): string[] {
+      const navItems = ['Application', 'Virtual Machine', 'Container Runtime', 'Kubernetes'];
+
+      return navItems.filter(x => this.isPlatformWindows && x !== 'Virtual Machine');
```

Comment:
The filter feels icky to me, but I'm not sure if my alternative is actually better, or just over-complicated…
```suggestion
      return ([
        ['Application', true],
        ['Virtual Machine', !this.isPlatformWindows],
        ['Container Runtime', true],
        ['Kubernetes', true],
      ] as const).filter(x => x[1]).map(x => x[0]);
```

Reply:
I think this gets simplified if we move WSL integrations out of the tab.